client/my-sites/plan-features-2023-grid/components/storage-add-on-dropdown.tsx

```diff
@@ -0,0 +1,59 @@
+import { CustomSelectControl } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { PlanSelectedStorage } from '..';
+import { PlanProperties } from '../types';
+import { getStorageStringFromFeature } from '../util';
+
+type StorageAddOnDropdownProps = {
+	planProperties: PlanProperties;
+	selectedStorage: PlanSelectedStorage;
+	setSelectedStorage: ( selectedStorage: PlanSelectedStorage ) => void;
+};
+
+export const StorageAddOnDropdown = ( {
+	planProperties,
+	selectedStorage,
+	setSelectedStorage,
+}: StorageAddOnDropdownProps ) => {
+	const { planName, storageOptions, storageFeatures } = planProperties;
+	const translate = useTranslate();
+
+	// TODO: Consider transforming storageOptions outside of this component
+	const selectControlOptions = storageOptions.reduce(
+		( acc: { key: string; name: string }[], storageOption ) => {
+			const storageString = getStorageStringFromFeature( storageOption );
+
+			// Only show storage options that have a string to display
```

Comment:
The comment seems a bit redundant. It's pretty obvious what's happening

Reply:
Removed in [64ae571](https://github.com/Automattic/wp-calypso/pull/79041/commits/64ae5719dcf7da33be4748fb23b0233291170cb2)